scripts/src/test/bash/functions-test

```diff
@@ -3,6 +3,21 @@
 function doCommandTest()  {
   cd ../test-setup
   CLI="${PWD}/scripts/devon"
-  "${CLI}" "${1}" setup
+  echo yes | "${CLI}" "${1}" setup
```

Comment:
can you simply run in batch mode (`--batch` or `-b`) instead?
FYI: I have meanwhile created #935 and implemented it already.

Reply:
I just updated this so I can approve now...